src/Microsoft.DotNet.Wpf/src/PresentationFramework/Resources/xlf/Strings.zh-Hans.xlf

```diff
@@ -4775,7 +4775,7 @@ Do you want to replace it?</source>
       </trans-unit>
       <trans-unit id="ScrollBar_ContextMenu_Bottom">
         <source>Bottom</source>
-        <target state="translated">底部</target>
+        <target state="translated">底</target>
```

Comment:
I do not think it is a good change.

Reply:
It seems related to https://github.com/dotnet/wpf/pull/4407#issuecomment-837452127.

Reply:
These were reverted henceforth resolving the conversation